From 7d72ab2c8127e6728b60ffdc63af50e522366428 Mon Sep 17 00:00:00 2001
From: Shohei YOSHIDA <shohei.yoshida@dena.com>
Date: Thu, 19 Jan 2017 14:58:10 +0900
Subject: Remove alpha version test

version.pm changed alpha version specification so that its test
makes no sense with latest version.pm

Bug: https://bugs.gentoo.org/623102
Bug: https://rt.cpan.org/Ticket/Display.html?id=112501
---
 MANIFEST                   | 1 -
 t/data/Perl/04.pm          | 5 -----
 t/version_from_file_perl.t | 4 ++--
 3 files changed, 2 insertions(+), 8 deletions(-)
 delete mode 100644 t/data/Perl/04.pm

diff --git a/MANIFEST b/MANIFEST
index b9b3cc1..a13981a 100644
--- a/MANIFEST
+++ b/MANIFEST
@@ -34,7 +34,6 @@ shipit
 t/data/Perl/01.pm
 t/data/Perl/02.pm
 t/data/Perl/03.pm
-t/data/Perl/04.pm
 t/data/Perl/05.pm
 t/data/Perl/06.pm
 t/data/Perl/07.pm
diff --git a/t/data/Perl/04.pm b/t/data/Perl/04.pm
deleted file mode 100644
index 1744dd1..0000000
--- a/t/data/Perl/04.pm
+++ /dev/null
@@ -1,5 +0,0 @@
-package something;
-
-use version; our $VERSION = qv('v1.5_0');
-
-1;
diff --git a/t/version_from_file_perl.t b/t/version_from_file_perl.t
index 286930a..c85d6f7 100644
--- a/t/version_from_file_perl.t
+++ b/t/version_from_file_perl.t
@@ -12,7 +12,7 @@ ok($proj);
 
 my $basedir = catfile(qw(t data Perl));
 
-for my $filenum ( 1 .. 7 ) {
+for my $filenum ( 1 .. 3, 5 .. 7 ) {
     my $origfile = catfile( $basedir, sprintf( '%02d.pm', $filenum ) );
 
     # check we read in the version correctly
@@ -33,7 +33,7 @@ for my $filenum ( 1 .. 7 ) {
 # cleanup
 END {
     eval {
-        for ( 1 .. 7 )
+        for ( 1 .. 3, 5 .. 7 )
         {
             unlink catfile( $basedir, sprintf( '%02d.pm.new', $_ ) );
         }
-- 
2.15.1

